android/sdk/src/main/java/org/jitsi/meet/sdk/ReactInstanceManagerHolder.java

```diff
@@ -24,6 +24,7 @@
 import com.facebook.react.bridge.ReactContext;
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.common.LifecycleState;
+import org.jitsi.meet.sdk.dropbox.Dropbox;
```

Comment:
@hristoterezov, for consistency please specify the full name of the class at its use site, don't import it and then use the short name. It's going to look something like:

````java
new org.jitsi.meet.sdk.dropbox.Dropbox(reactContext),
new org.jitsi.meet.sdk.invite.InviteModule(reactContext),
new org.jitsi.meet.sdk.net.NAT64AddrInfoModule(reactContext)
````